### PR TITLE
Add Cognni's Analytic Rule Templates

### PR DESCRIPTION
**Pull request by adirDev**
Fixes #

## Proposed Changes

  -
  -
  -

### CODE REVIEW

**Comment by preetikr** [comment]
@v-jayakal - these might need to be moved to solutions and please help review metadata part of detections. Thanks.
@shainw - please review the detections. Thanks.

**Comment by oshezaf** [comment]
@shainw @preetikr @Ronmarsiano : this PR should really be implemented using a single rule and the new dynamic alert details.

**Review by v-admahe — APPROVED** [review]

**Comment by Lodewyk-Git** [comment]
Hi @oshezaf are there any publicly available documentation or posts associated with the new dynamic alert style?

**Comment by oshezaf** [comment]
@Ronmarsiano : where can @Lodewyk-Git find documentation of custom alert details? especially how to represent it in the YAML? 

@Lodewyk-Git : I will chat with Ron tomorrow.

**Comment by v-admahe** [comment]
@shainw We have check the metadata of analytic rule and it looks good to us.

Thanks,
Aditi Maheshwari

**Comment by oshezaf** [comment]
Hi @Lodewyk-Git : I updated the wiki to [document how to use dynamic alert details in YAML](https://github.com/Azure/Azure-Sentinel/wiki/Query-Style-Guide#alert-details)

**Comment by shainw** [comment]
@v-admahe - I have approved, but I think many of these could be collapsed into 1 by using the [Customize alert details feature](https://docs.microsoft.com/en-us/azure/sentinel/customize-alert-details).  Example in the [Wiki](https://github.com/Azure/Azure-Sentinel/wiki/Query-Style-Guide#alertdetailsoverride).

Please look at doing another PR to collapse these using the following, where we have the same query but a slightly different clause in each:

### "Cognni Incidents for {{sevType}} Sensitivity **{{infoType}}**"
we can simply assign the "informationType_s" column to the {{infoType}} variable and the "Severity" column to the {{sevType}} variable.

Basically, this moves us to1 query, but 15 detections.

**Review by shainw — APPROVED** [review]

**Comment by v-jayakal** [comment]
@adirDev, please consider directions provided by @shainw for efficient solution for analytic rules and raise another PR. We are merging these for now.

**Comment by adirDev** [comment]
@v-jayakal, I will consider the directions provided by @shainw for efficient solution and will raise another PR. 
Thank you
